powerhome/playbook
# [PLAY-681] Change Star Rating Kit from JSX to TSX

#### Screens

![screenshot-127 0 0 1_3000-2023 04 05-13_42_01](https://user-images.githubusercontent.com/38965626/230160775-5186a936-494f-437d-b9a9-10775fde9dde.png)

#### Breaking Changes

I don;t think so :3333

#### Runway Ticket URL

https://nitro.powerhrg.com/runway/backlog_items/PLAY-681

#### How to test this

This is not currently used in nitro, but you can go to `/kits/star_rating/react` and check to make sure it works
#### Checklist:
